node/src/components/consensus.rs

```diff
@@ -220,23 +227,45 @@ where
                 sender,
                 proto_block,
             } => handling_es.handle_invalid_proto_block(era_id, sender, proto_block),
-            Event::GetValidatorsResponse {
+            Event::CreateNewEra {
                 block_header,
+                booking_block_hash,
+                key_block_seed,
                 get_validators_result,
-            } => match get_validators_result {
-                Ok(Some(result)) => {
-                    handling_es.handle_get_validators_response(*block_header, result)
-                }
-                result => {
+            } => {
+                let booking_block_hash = booking_block_hash.unwrap_or_else(|| {
                     error!(
-                        ?result,
-                        "get_validators in era {} returned an error: {:?}",
-                        block_header.era_id(),
-                        result
+                        "could not get the booking block hash for era {}",
+                        block_header.era_id().successor()
```

Comment:
A more useful log message would be one containing the hash of the block we asked for, rather than the ID of an era we want to create.